src/main/g8/build.sbt

```diff
@@ -1,5 +1,8 @@
+import org.typelevel.sbt.tpolecat.*
+
 ThisBuild / organization := "com.example"
 ThisBuild / scalaVersion := $if(scala3.truthy)$"3.4.0"$else$"2.13.13"$endif$
+ThisBuild / tpolecatDefaultOptionsMode := VerboseMode;
```

Comment:
Can we add here the aforementioned comment? Something like `Remember to set "SBT_TPOLECAT_CI" in your CI to enable fatal-warnings and yada yada`?

Reply:
ah sorry, my bad

Reply:
```suggestion
ThisBuild / tpolecatDefaultOptionsMode := VerboseMode
```